tests/unit/build_providers/test_base_provider.py

```diff
@@ -506,10 +505,11 @@ def test_setup_snapcraft_with_base(self):
             [
                 call(snap_name="snapd"),
                 call(snap_name="core18"),
+                call(snap_name="core20"),
```

Comment:
Should we also run this test with base core20 to verify if the base snap is not injected twice?

Reply:
we already have that test :-)

These tests will mostly go away or change when we switch to build providers too